contrib/inventory/puppetdb.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+#
+# Ansible inventory script based on puppetdb.
+#
+# Author: Jan Collijs
+# Source: https://github.com/visibilityspots/ansible-puppet-inventory
+
+# If you want to use the api version 4 you have to install the pypuppetdb
+# library made by https://github.com/NeCTAR-RC/pypuppetdb from source untill
+# it got merged https://github.com/puppet-community/pypuppetdb/pull/34
```

Comment:
It's not merged, but over the years it got implemented natively in puppetdb itself. Back in the days when I wrote this script it wasn't available yet in the upstream code base of puppetdb.